src/services/redisSessionStorage.ts

```diff
@@ -317,6 +293,46 @@ export default class RedisSessionStorage extends RedisStorageUtils
     );
   }
 
+  /*
+  * Return the promises needed to store session info and 
+  * update session info set
+  */
+  private saveSessionInfo(user: User): Promise<Either<Error, boolean>> {
+    const newSessionInfo: SessionInfo = {
```

Comment:
this method should probably accept just a `SessionInfo`, not a `User` - functions should require the smallest set of information needed

Reply:
Is required even `user.fiscal_code` at [this](https://github.com/teamdigitale/io-backend/pull/444/files#diff-178a7f5864283f425f4e6d8d513710cfR322) row.

Reply:
Should I leave the code unchanged? Or I change the function with 2 parameters: `SessionInfo` and `FiscalCode`?

Reply:
yes two params is better

Reply:
fixed in 1bd3e55